### PR TITLE
[TIMOB-18287] Fix PowerShell Input Text Invisible

### PR DESCRIPTION
**Pull request by dawsontoth**
https://jira.appcelerator.org/browse/TIMOB-18287

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ var fs = require('fs'),
 	path = require('path'),
 	colors = require('colors'),
 	common = require('./common'),
+	// Windows PowerShell translates magenta to the same blue as the background. So use white, instead.
+	inputAndMaskColor = process.title.indexOf('PowerShell') >= 0 ? 'white' : 'magenta',
```
Comment (cb1kenobi) [inline lib/index.js]: It's crazy that `process.title` is "Windows PowerShell". In the normal command prompt, the value is "Command Prompt - node  test.js".

What do you think about adding `process.platform === 'win32'` to the check?

Comment (dawsontoth) [inline lib/index.js]: Yeah, I agree -- the title is a bit crazy, but I'm not sure how else to detect it. I looked around for a bit and didn't find other differences. Maybe there is a better way? Anyway, yeah, adding the process.platform check could help.

**Comment by dawsontoth** [comment]
@cb1kenobi adjusted the PR, by the way. Look ok now?

**Comment by cb1kenobi** [comment]
Looks good to me. I looked into alternate ways of detecting if being run from PowerShell and there's really no other way than spawning something. So, `process.title` it is! Thanks for the PR! :)

**Pull request merged**